data.json

```diff
@@ -115,7 +115,9 @@
       "go1.11.13.linux-amd64.tar.gz",
       "go1.12.17.linux-amd64.tar.gz",
       "go1.14.2.linux-amd64.tar.gz",
+      "go1.15.9.linux-amd64.tar.gz",
```

Comment:
This is an unfortunate side effect of requiring test assets to be declared. Testing version expansion this way requires the file to be changed whenever new Go versions are added and will require downloading a couple hundred megabytes more before tests can run.

Reply:
Yeah not ideal, though think still worth adding these tests for now and fixing the way tests work later. (Ideally this repo would use Hatchet tests for much of this anyway.)